suse_migration_services/units/reboot.py

```diff
@@ -37,3 +55,10 @@ def main():
         Command.run(
             ['reboot', '-f']
         )
+
+
+def check_failed_migration():
+    has_failed = Command.run(
+        ['systemctl', 'is-failed', 'suse-migration']
+    )
+    return has_failed == 'failed'
```

Comment:
I think this can't work. Command.run returns a tuple (see command.py):

```python
return command_type(
    output=output.decode(),
    error=error.decode(),
    returncode=process.returncode
)
```

You want to compare the stdout output. So the code should be written like this:

```python
systemctl_command = Command.run(....)
return True if systemctl_command.output == 'failed' else False
```

As your unit tests are passing I assume the Command mock is also wrong